simple_history/models.py

```diff
@@ -307,5 +318,5 @@ def __init__(self, model):
 
     def __get__(self, instance, owner):
         values = (getattr(instance, f.attname)
-                  for f in self.model._meta.fields)
+                  for f in HistoricalRecords.get_fields(self.model))
```

Comment:
Could we change this one to use `super()`? That would allow others to override `get_fields()` without needing to override `__get__()` as well.